lib/tasks/argo.rake

```diff
@@ -18,6 +18,7 @@ end
 task :default => [:ci]
 
 task :ci do
+  raise 'NEVER RUN CI ON PROD' if ENV['RAILS_ENV'] =~ /production/i
   ENV['RAILS_ENV'] = 'test'
```

Comment:
what about replacing the proposed exception and the existing `ENV['RAILS_ENV'] = 'test'` lines with something like `raise 'NEVER run CI outside of test!' unless ENV['RAILS_ENV'] == 'test'`?

that seems even more paranoid than the currently proposed change, and doesn't lead to a false sense of security or some weird state by resetting an environment var in a way that might not even have the intended effect.  my reasoning is that i've seen rake tasks fail to execute because the config/environments/ files that were being loaded had some error, preventing Rails itself from loading.  resetting RAILS_ENV here will not cause those configs to be reloaded, and the loading of those configs determines more meaningful environment specific behavior than anything else.